### PR TITLE
[REVIEW] Remove `cuda 10.x` related files

### PR DESCRIPTION
**Pull request by galipremsagar**
This PR removes cuda `10.x` related files and updates the docker script(conda version) and its associated readme.

### CODE REVIEW

**Review by ajschmidt8 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -320,7 +320,7 @@ flag. Below is a list of the available arguments and their purpose:
 | `PYARROW_VERSION` | 1.0.1 | Not supported | set pyarrow version |
 | `CMAKE_VERSION` | newest | >=3.14 | set cmake version |
 | `CYTHON_VERSION` | 0.29 | Not supported | set Cython version |
-| `PYTHON_VERSION` | 3.7 | 3.8 | set python version |
+| `PYTHON_VERSION` | 3.8 | 3.9 | set python version |
```
Comment (kkraus14) [inline CONTRIBUTING.md:323]: I think we still support 3.7, no?

Comment (galipremsagar) [reply CONTRIBUTING.md]: Sorry, I was under the assumption we dropped 3.7 support. Correct this.

**Review by kkraus14 — APPROVED** [review]

**Comment by galipremsagar** [comment]
rerun tests

**Comment by kkraus14** [comment]
rerun tests

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -309,9 +309,9 @@ flag. Below is a list of the available arguments and their purpose:
 
 | Build Argument | Default Value | Other Value(s) | Purpose |
 | --- | --- | --- | --- |
-| `CUDA_VERSION` | 10.0 | 10.1, 10.2 | set CUDA version |
-| `LINUX_VERSION` | ubuntu16.04 | ubuntu18.04 | set Ubuntu version |
-| `CC` & `CXX` | 5 | 7 | set gcc/g++ version; **NOTE:** gcc7 requires Ubuntu 18.04 |
+| `CUDA_VERSION` | 11.0 | 11.1, 11.2.2 | set CUDA version |
```
Comment (karthikeyann) [inline CONTRIBUTING.md:312]: There are few instances of 10.0 usage in this file.
https://github.com/galipremsagar/cudf/blob/remove_10.x/CONTRIBUTING.md#L163
https://github.com/galipremsagar/cudf/blob/remove_10.x/CONTRIBUTING.md#L285

File: Dockerfile [hunk 3]
```diff
@@ -70,14 +72,9 @@ RUN if [ -f /cudf/docker/package_versions.sh ]; \
...
+    cd /cudf/ && \
+    ./build.sh libcudf
 
 # cuDF build/install
 RUN source activate cudf && \
-    cd /cudf/python/cudf && \
-    python setup.py build_ext --inplace && \
-    python setup.py install && \
-    python setup.py install
+    cd /cudf/ && ./build.sh cudf
```
Comment (karthikeyann) [inline Dockerfile:80]: Why not build using single command?

```suggestion
    cd /cudf/ && \
    ./build.sh libcudf cudf
```

**Review by karthikeyann — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (galipremsagar) [reply CONTRIBUTING.md:312]: Done

File: Dockerfile (hunk 3 above)
Comment (galipremsagar) [reply Dockerfile]: Merged into a single command.

**Review by mike-wendt — APPROVED** [review]

**Comment by galipremsagar** [comment]
rerun tests

**Comment by kkraus14** [comment]
@gpucibot merge

**Pull request merged**